types/database/raw_values.d.ts

```diff
@@ -29,8 +29,10 @@ type Draft = {
 type ScheduledPost = Draft & {
     id: string;
     scheduled_at: number;
-    processed_at: number;
-    error_code: string;
+    processed_at?: number;
+    error_code?: string;
+    priority?: PostPriority;
+    file_ids?: string[];
```

Comment:
Don't we have file ids and priority for drafts already?

Reply:
@larkox we do, buut they are saved in draft.metadata. Unfortunately the structure of draft on webapp and mobile is out of sync. The API is designed from webapp's structure which sends priority and file IDs outside metadata, not inside. So, while drafts on mobile have priority and file IDs in metadata, to serve the scheduled post API, I need them outside.

Reply:
@rahimrahman yeah I've removed the one from `raws.d.ts` now.